solution/zenko-version-cr.yaml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
This manifest will not be deployed by the Solution framework, what's the purpose of it ? Providing an example shipped with the Solution ?

Reply:
This file was removed